hubspot/serializers.py

```diff
@@ -43,6 +45,15 @@
 }
 
 
+# This was taken from xpro
+ORDER_STATUS_MAPPING = {
+    Order.FULFILLED: "processed",
+    Order.FAILED: "checkout_completed",
+    Order.CREATED: "checkout_completed",
+    Order.REFUNDED: "processed",
+}
+
+
 class HubspotProfileSerializer(serializers.ModelSerializer):
```

Comment:
nit: rename to `HubspotContactsSerializer` to be consistent in naming, since we seem to be referring to hubspot objects here...